frontend/src/pulldasher.tsx

```diff
@@ -1,15 +1,16 @@
-import * as React from 'react';
 import usePullsState from './pulls-state';
 import PullsContext from './pulls-context';
-import { Pull } from './types';
+import { Pull } from './pull';
 import Navbar from './navbar';
-import CRColumn from './cr-column';
+import { Column } from './column';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 
 export default function() {
    const pulls: Pull[] = usePullsState();
+   const pullsDevBlocked = pulls.filter(pull => pull.isDevBlocked());
+   const pullsNeedingCR = pulls.filter(pull => pull.isCrDone());
```

Comment:
I think this is backwards
```suggestion
   const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone());
```